.github/workflows/tests-template.yml

```
@@ -15,6 +15,8 @@ permissions: read-all
 
 jobs:
   test-linux:
+    # this is to prevent arm64 jobs from running at forked projects
+    if: ${{ github.repository == 'etcd-io/bbolt' || startsWith(inputs.runs-on, 'ubuntu') || startsWith(inputs.runs-on, 'windows') }}
```

This template isn't used by Windows workflow yet.
```suggestion
    if: ${{ github.repository == 'etcd-io/bbolt' || startsWith(inputs.runs-on, 'ubuntu') || startsWith(inputs.runs-on, 'windows') }}
```
```suggestion
    if: ${{ github.repository == 'etcd-io/bbolt' || startsWith(inputs.runs-on, 'ubuntu') }}
```

---

Good point. Committed and pushed.